ui/public/i18n/language.json

```
@@ -513,6 +513,7 @@
         "signature": "Enable DKIM signing for outgoing messages to prevent spoofing.",
         "acls": "Type to search a user or group for ACLs.",
         "bayes_title": "Bayes filter training status.",
+        "virus_scan_only_attachments": "Scan mime parts of attachments seperately, otherwise the complete mail will be transfered to Antivirus Scanner.",
```

Antivirus is spelled in lowercase in other labels, also fix a couple of typos

```suggestion
        "virus_scan_only_attachments": "Scan mime parts of attachments separately, otherwise the complete mail will be transferred to the antivirus.",
```

(Feel free to commit and rebase)